code/game/machinery/constructable_frame.dm

```diff
@@ -754,6 +754,15 @@ obj/item/weapon/circuitboard/rdserver
 							"/obj/item/weapon/stock_parts/matter_bin" = 1,
 							"/obj/item/weapon/stock_parts/capacitor" = 2)
 
+/obj/item/weapon/circuitboard/box_cloner
+	name = "Circuit Board (Box Cloner)"
+	build_path = "/obj/machinery/egg_incubator/box_cloner"
+	origin_tech = "illegal=3"
```

Comment:
@Konater The designs aren't very expensive - you can get most of the useful stuff after a gold/silver/uranium shipment, don't even have to wait for diamonds..

@Kurfursten Yes, but the tesla relays are, as I already said, locked behind research and materials - it's impossible to get them earlier than the miners come in with mats.

Also I'm pretty sure the illegal tech is called "syndicate", which is one of the pains of doing origin tech by string because you can't use defines for initial values.